src/doc/integrating_telio.md

```diff
@@ -1,5 +1,891 @@
 # Documentation For Integrating Telio Into Other Languages
 
-<!-- TODO: This page is a placeholder and will be populated by upcoming PR's when the actual specification will be more clear -->
+## Integration matrix
+
+| Feature/App                | TCLI | Telio-Apple | Android App | Windows App | Linux App | 
```

Comment:
No2 I consider resolved when the note is added that it is really just internal doc

Reply:
I had a quick chat with @packgron, this table was useful in the past but can now be removed, so that's what I'll do. I'm resolving the other comments related to the table as well